apps/account-api/src/services/accounts.service.ts

```diff
@@ -54,12 +54,12 @@ export class AccountsService {
   }
 
   async getSIWFConfig(): Promise<WalletLoginConfigResponseDto> {
-    const { frequencyHttpUrl, siwfUrl }: IAccountApiConfig = this.apiCOnf;
+    const { siwfNodeRpcUrl, siwfUrl }: IAccountApiConfig = this.apiCOnf;
     const { providerId } = this.blockchainConf;
     return {
       providerId: providerId.toString(),
       siwfUrl: siwfUrl.toString(),
-      frequencyRpcUrl: frequencyHttpUrl.toString(),
+      frequencyRpcUrl: siwfNodeRpcUrl.toString(),
```

Comment:
nit: Do we need to change `WalletLoginConfigResponseDto`  also or since that would break the contract we are just going to assume the user knows!?